src/main/java/core/basesyntax/figures/Square.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.Figure;
+
+public class Square extends Figure {
+    private static final String type = "square";
+    private final int side;
+
+    public Square(int side, String color) {
+        this.side = side;
+        this.color = color;
+    }
+
+    @Override
+    public double getArea() {
+        return (double) side * side;
```

Comment:
There is no need to cast 'side' to a double explicitly since multiplying two ints will result in an int, which will then be implicitly cast to a double when returned.